desktop/package/linux/Haveno.AppDir/Haveno.desktop

```diff
@@ -0,0 +1,16 @@
+# If you want Haveno to appear in a Linux app launcher ("start menu"), install this by doing:
```

Comment:
Is this needed? If the AppImage comes with this file, wouldn't AppImageLauncher be responsible for installing the desktop file? I don't think a user would ever see this.

Reply:
True, I'll remove this.